auth-lobby/src/env.ts

```diff
@@ -0,0 +1,9 @@
+if (typeof process.env.REACT_APP_RELAY_HOST !== 'string') {
+  throw new Error('ENV: No relay host provided')
+}
+export const RELAY_HOST = process.env.REACT_APP_RELAY_HOST
+// @TODO: this is temporary for dev purposes
+if (typeof process.env.REACT_APP_ROOT_USER !== 'string') {
```

Comment:
Is this check correct? Shouldn't it be looking at RELAY_HOST?

Reply:
May just be that the error below is wrong

Reply:
ah yup just the error is wrong. thx!